chain/events/filter/index.go

```diff
@@ -520,6 +640,61 @@ func (ei *EventIndex) Close() error {
 	return ei.db.Close()
 }
 
+func (ei *EventIndex) SubscribeUpdates() (uint64, chan *EventIndexUpdate) {
```

Comment:
perhaps you should just return an `Unsubscribe()` or `Cancel()` function from this instead of a uint64 so you don't need a second method for `UnsubscribeUpdates`, and you also don't allow for one caller to cancel the subscription of another

Reply:
Done.